yaml_files/cuba.yml

```diff
@@ -565,3 +555,59 @@ CUBA_KEYS:
     definition: Thermodynamic ensemble
     shape: [1]
     type: string
+
+  LATTICE_UC_ABC:
+    definition: Lattice translation vectors lengths (a, b, c) of an atomic unit cell
+    shape: [3]
+    type: double
+
+  LATTICE_UC_ANGLES:
```

Comment:
Please remove UC, instead its LATTICE_ANGLES or expand UC to UNIT_CELL. Please note that this does not give any information of whether this is a primitive unit cell or not! Alternatively we can think of a CRYSTAL_ LATTICE as a data structure in itself, that has a unit cell that has vectors and angles and symmetry etc.